src/main/flight/pid.c

```diff
@@ -445,12 +445,11 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     horizonFactorRatio = (100 - pidProfile->horizon_tilt_effect) * 0.01f;
     maxVelocity[FD_ROLL] = maxVelocity[FD_PITCH] = pidProfile->rateAccelLimit * 100 * dT;
     maxVelocity[FD_YAW] = pidProfile->yawRateAccelLimit * 100 * dT;
-    ITermWindupPointInv = 0.0f;
```

Comment:
I just would add this line with the default 1.0f value and remove that else block.

Reply:
Did you see Mike's earlier comment about needing to keep the else block?

Reply:
Thats why i said to add this removed line with the new default value. It does exact the thing as the else without the need of having it

Reply:
```
variable = defaultValue;
if (condition) {
  variable = newValue;
}
```
Is preferable over an `else` block

Reply:
duh sorry now I get it!  thank you.  final camelCase stuff done too, hopefully

Reply:
@DieHertz: I don't care either way, but since the function can be called at runtime it should be one of the two. Out of interest, what is the reason for this preference? The `else` variant definitely wins out in terms of maintainability.